azure-pipelines.yml

```diff
@@ -3,14 +3,9 @@
 # Add steps that build, run tests, deploy, and more:
 # https://aka.ms/yaml
 
-trigger:
```

Comment:
we use schedules in pipelines/official_build.yml
triggers is for CI trigger: when you push an update to the specified branches or you push specified tags.
schedules is for daily build.

Reply:
keep this, this is for the quick build.

Reply:
changed. but I don't understand. PR build is pre-build. CI build is post-build. Isn't it duplicated?

Reply:
for pr build, we do it for rcache, this allows us to get a quick build.